0_RootFS/Rootfs/build_tarballs.jl

```diff
@@ -117,7 +117,7 @@ sources = [
               "c68e441d2b93074b01ea193cb17e944ed751750f"), # v2.54
     # As is patchelf
     GitSource("https://github.com/NixOS/patchelf.git",
-              "99c24238981b7b1084313aca8f5c493bb46f302c"), # v0.18.0
+              "bf3f37ec29edcdb3e2a163edaf84aeece39f8c9d"), # v0.14.3
```

Comment:
It'd be good to leave a comment to explain we don't want to upgrade patchelf unless there's a compelling and proved reason to do it because of previous problems we experienced, with references to the error message and a link to discussion.